diff --git a/libqalculate/Calculator.cc b/libqalculate/Calculator.cc
index 6f33ad7..b76d67f 100644
--- a/libqalculate/Calculator.cc
+++ b/libqalculate/Calculator.cc
@@ -208,7 +208,7 @@ Calculator::Calculator() {
 	priv->can_plot = -1;
 	priv->concise_uncertainty_input = false;
 	priv->fixed_denominator = 2;
-	priv->definitions_locale_set = false;
+	priv->definitions_locale_set = true;
 
 #ifdef HAVE_ICU
 	UErrorCode err = U_ZERO_ERROR;
@@ -484,7 +484,7 @@ Calculator::Calculator(bool ignore_locale) {
 	priv->can_plot = -1;
 	priv->concise_uncertainty_input = false;
 	priv->fixed_denominator = 2;
-	priv->definitions_locale_set = ignore_locale;
+	priv->definitions_locale_set = true;
 
 #ifdef HAVE_ICU
 	UErrorCode err = U_ZERO_ERROR;
diff --git a/libqalculate/Makefile.am b/libqalculate/Makefile.am
index 3729a26..6d45f69 100644
--- a/libqalculate/Makefile.am
+++ b/libqalculate/Makefile.am
@@ -56,4 +56,5 @@ libqalculate_la_LIBADD = \
 AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
+	-std=c++11 \
 	@LIBCURL_CFLAGS@  @LIBXML_CFLAGS@  @ICU_CFLAGS@
diff --git a/src/Makefile.am b/src/Makefile.am
index 51e9cae..15b4b79 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -6,6 +6,7 @@ AM_CPPFLAGS = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/libqalculate \
+	-std=c++11 \
 	@LIBCURL_CFLAGS@ @LIBXML_CFLAGS@ @ICU_CFLAGS@
 
 bin_PROGRAMS = @QALCULATE_TEXT@
diff --git a/po-defs/remove-untranslated.cc b/po-defs/remove-untranslated.cc
index 6e35b59..02feef2 100644
--- a/po-defs/remove-untranslated.cc
+++ b/po-defs/remove-untranslated.cc
@@ -89,11 +89,9 @@ int main(int argc, char *argv[]) {
 				}
 				bool omit = msgid.find("-r:") != string::npos;
 				bool hyphen = msgid.find("r:") != string::npos;
-				if(!msgid.empty() && (msgstr == msgid || msgstr == "-" || msgstr == "+")) {
+				if(true) {
 					if(variant == 3 || variant == 4) cout << msgstr << endl;
-					if(msgstr == "+" || (!omit && !hyphen)) {sout = "msgstr \""; sout += msgid; sout += "\"";}
-					else if(!omit) sout = "msgstr \"-\"";
-					else sout = "msgstr \"\"";
+					sout = "msgstr \"\"";
 				} else {
 					size_t i = 0, i2 = 0;
 					while(true) {
