diff --git a/libqalculate/Calculator-definitions.cc b/libqalculate/Calculator-definitions.cc
index d53272a..2ece786 100644
--- a/libqalculate/Calculator-definitions.cc
+++ b/libqalculate/Calculator-definitions.cc
@@ -736,12 +736,7 @@ int Calculator::loadDefinitions(const char* file_name, bool is_user_defs, bool c
 	if(clocale) locale = clocale;
 #endif
 
-	if(b_ignore_locale || locale == "POSIX" || locale == "C") {
-		locale = "";
-	} else {
-		size_t i = locale.find('.');
-		if(i != string::npos) locale = locale.substr(0, i);
-	}
+	locale = "";
 
 	int fulfilled_translation = 0;
 	string localebase;
diff --git a/libqalculate/Makefile.am b/libqalculate/Makefile.am
index 3729a26..6d45f69 100644
--- a/libqalculate/Makefile.am
+++ b/libqalculate/Makefile.am
@@ -56,4 +56,5 @@ libqalculate_la_LIBADD = \
 AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
+	-std=c++11 \
 	@LIBCURL_CFLAGS@  @LIBXML_CFLAGS@  @ICU_CFLAGS@
diff --git a/src/Makefile.am b/src/Makefile.am
index 51e9cae..15b4b79 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -6,6 +6,7 @@ AM_CPPFLAGS = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/libqalculate \
+	-std=c++11 \
 	@LIBCURL_CFLAGS@ @LIBXML_CFLAGS@ @ICU_CFLAGS@
 
 bin_PROGRAMS = @QALCULATE_TEXT@
