diff --git a/libqalculate/Calculator-definitions.cc b/libqalculate/Calculator-definitions.cc
index 2e2a979..0fb2a18 100644
--- a/libqalculate/Calculator-definitions.cc
+++ b/libqalculate/Calculator-definitions.cc
@@ -700,12 +700,7 @@ int Calculator::loadDefinitions(const char* file_name, bool is_user_defs, bool c
 	if(clocale) locale = clocale;
 #endif
 
-	if(b_ignore_locale || locale == "POSIX" || locale == "C") {
-		locale = "";
-	} else {
-		size_t i = locale.find('.');
-		if(i != string::npos) locale = locale.substr(0, i);
-	}
+	locale = "";
 
 	int fulfilled_translation = 0;
 	string localebase;
diff --git a/libqalculate/Makefile.am b/libqalculate/Makefile.am
index f7865ec..609d4f5 100644
--- a/libqalculate/Makefile.am
+++ b/libqalculate/Makefile.am
@@ -63,4 +63,5 @@ libqalculate_la_LIBADD = \
 AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
+	-std=c++11 \
 	@LIBCURL_CFLAGS@  @LIBXML_CFLAGS@  @ICU_CFLAGS@ @GIO_CFLAGS@
diff --git a/src/Makefile.am b/src/Makefile.am
index f65d2b2..32e7cce 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -6,6 +6,7 @@ AM_CPPFLAGS = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/libqalculate \
+	-std=c++11 \
 	@LIBCURL_CFLAGS@ @LIBXML_CFLAGS@ @ICU_CFLAGS@ @GIO_CFLAGS@
 
 bin_PROGRAMS = @QALCULATE_TEXT@
